mythril/laser/smt/bitvec.py

```diff
@@ -254,6 +286,10 @@ def _arithmetic_helper(a: BitVec, b: BitVec, operation: Callable) -> BitVec:
     return BitVec(raw, annotations=union)
 
 
+def LShR(a: BitVec, b: BitVec):
```

Comment:
The annotations should be unioned and passed to the new object

Reply:
_arthmetic_helper does that right?